### PR TITLE
feat: reworked the docs for dev deployment

### PR DESCRIPTION
**Pull request by andreas-kupries**
Doc only changes. No functional changes. CI troubles can be ignored.

Rephrased to impersonal voice, rearranged and explained the special commands needed, and added notes on the latest post-deployment actions of install.

### CODE REVIEW

File: docs/development.md [hunk 1]
```diff
@@ -36,53 +36,95 @@ This is building Epinio for linux on amd64 architecture. If you are on a
...
 
-
 ### Installing Epinio
 
 You can have a look at [the dedicated document](/docs/install.md) for cluster
 specific instructions, but generally this should be sufficient to get you running:
 
 ```
-./dist/epinio-linux-amd64 install
+export EPINIO_DONT_WAIT_FOR_DEPLOYMENT=1
```
Comment (thardeck) [inline docs/development.md:45]: I suppose it is irrelevant if we use `true` or `1`?

File: docs/development.md [hunk 2]
```diff
@@ -36,53 +36,95 @@ This is building Epinio for linux on amd64 architecture. If you are on a
...
+be adjusted accordingly.
+
+After making changes to the binary simply invoking `make
+patch-epinio-deployment` again will upload the changes into the
+running cluster.
+
+Another thing `epinio install` does after deploying all components is
+the creation and targeting of a standard organization, `workspace`.
+
+Wtih the failing server component these actions will fail, making the
```
Comment (thardeck) [inline docs/development.md:107]: Typo `With`

**Review by thardeck — CHANGES_REQUESTED** [review]

File: docs/development.md (hunk 1 above)
Comment (andreas-kupries) [reply docs/development.md:45]: Would have to check.

File: docs/development.md (hunk 2 above)
Comment (andreas-kupries) [reply docs/development.md:107]:
```suggestion
With the failing server component these actions will fail, making the
```

File: docs/development.md (hunk 1 above)
Comment (andreas-kupries) [reply docs/development.md:45]: Checked: Any non-empty string is sufficient.
https://github.com/epinio/epinio/blob/main/deployments/epinio.go#L195

**Review by thardeck — APPROVED** [review]

**Pull request merged**